tests/v2/validation/provisioning/rke1/README.md

```diff
@@ -67,6 +67,8 @@ provisioningInput:
         secretKey: ""
     retention: "72h"
     snapshot: false
+chartUpgrade:
```

Comment:
nit; can you leave a comment explaining this a bit more? I don't think this is immediately intuitive to someone new to this test.

Reply:
I agree I still don't understand why we need this new configuration.

Reply:
added a comment here, lmk if it is more clear. But for more context, this is the main feature of this PR - allowing the chart to be upgraded via automation. Before, we just installed the latest version of the chart by default in automation. If that's the case, we can't upgrade it later. So this option installs latest - 1 so that the chart can be upgraded in a later test.